src/handlers/triggerStream.js

```diff
@@ -7,7 +7,30 @@ import AWS from 'aws-sdk' ;
  * @param {import('serverless/plugins/aws/package/compile/events/apiGateway/lib/validate').ApiGatewayEvent} event 
  * @returns {JSON}
  */
+ let topic = process.env.customerCreatedSNS;
 exports.handler = async (event) => {
-    console.log(event);
-    console.log(event.Records[0]);
+    
+    // console.log(event);
+    console.log(event.Records[0].eventName);
+    let sns= new AWS.SNS(
+      {
+      endpoint:"http://127.0.0.1:4002",
+      region: "eu-west-1"
+    }
+    );
+    let resPub;
+    topic="arn:aws:sns:eu-west-1:123456789012:"+topic;
```

Comment:
Can it not be declared above in L10 as a const?

Side note: but use of const is preferred where possible so we are not mutating variables.